flink-streaming-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaShortRetentionTestBase.java

```diff
@@ -60,18 +64,33 @@
 	private static Properties standardProps;
 	private static ForkableFlinkMiniCluster flink;
 
+	@ClassRule
+	public static TemporaryFolder tempFolder = new TemporaryFolder();
+
+	protected static Properties secureProps = new Properties();
+
 	@BeforeClass
 	public static void prepare() throws IOException, ClassNotFoundException {
 		LOG.info("-------------------------------------------------------------------------");
 		LOG.info("    Starting KafkaShortRetentionTestBase ");
 		LOG.info("-------------------------------------------------------------------------");
 
+		Configuration flinkConfig = new Configuration();
+
 		// dynamically load the implementation for the test
 		Class<?> clazz = Class.forName("org.apache.flink.streaming.connectors.kafka.KafkaTestEnvironmentImpl");
 		kafkaServer = (KafkaTestEnvironment) InstantiationUtil.instantiate(clazz);
 
 		LOG.info("Starting KafkaTestBase.prepare() for Kafka " + kafkaServer.getVersion());
 
+		LOG.info("Runtype: {}", RunTypeHolder.get());
+		if(RunTypeHolder.get().equals(RunTypeHolder.RunType.SECURE)
+				&& kafkaServer.isSecureRunSupported()) {
+			SecureTestEnvironment.prepare(tempFolder);
+			SecureTestEnvironment.getSecurityEnabledFlinkConfiguration(flinkConfig);
```

Comment:
No. The passed config object will be populated with keytab and principal security configurations.